bindings/java/src/main/java/ethereum/ckzg4844/CKZG4844JNI.java

```diff
@@ -43,24 +43,36 @@ public static void loadNativeLibrary() {
     }
   }
 
-  /** Scalar field modulus of BLS12-381 */
-  public static final BigInteger BLS_MODULUS =
```

Comment:
I prefer to keep those public in order to use those in projects which use the library

Reply:
Hmm, protecting those constants was a conscious decision. It was an attempt to remove cryptographic details from the clients. In some cases cryptography code can be easy to mess up, and client devs might not be aware of the various cryptographic intricacies, so this change was done so that cryptography logic gets concentrated in the libraries.

How do you use these constants in Teku?

Reply:
We have a utility class which flattens a list of commitments/proofs for example and we calculate the expected size based on these constants e.g. https://github.com/Consensys/teku/blob/master/infrastructure/kzg/src/main/java/tech/pegasys/teku/kzg/CKZG4844Utils.java

We also use in tests https://github.com/Consensys/teku/blob/master/infrastructure/kzg/src/test/java/tech/pegasys/teku/kzg/CKZG4844Test.java

I suppose not an issue to replicate those in Teku internally in the module, but would lead to duplication.

Reply:
Stefan, we've decided to revert this change. So these constants are public again for all bindings. With the exception of the `BYTES_PER_G*` constants because `load_trusted_setup` takes a flat array of bytes now.